LocationList.py

```diff
@@ -1855,12 +1866,12 @@ def shop_address(shop_id, shelf_id):
     ("Gerudo Training Ground MQ Second Iron Knuckle Chest",          ("Chest",        0x0B,  0x12, None,                            'Arrows (10)',                           ("Gerudo Training Ground MQ", "Master Quest", "Chests",))),
     ("Gerudo Training Ground MQ Flame Circle Chest",                 ("Chest",        0x0B,  0x0E, None,                            'Small Key (Gerudo Training Ground)',    ("Gerudo Training Ground MQ", "Master Quest", "Chests",))),
     ("Gerudo Training Ground MQ Maze Right Central Chest",           ("Chest",        0x0B,  0x05, None,                            'Rupees (5)',                            ("Gerudo Training Ground MQ", "Master Quest", "Chests",))),
-    ("Gerudo Training Ground MQ Maze Right Side Chest",              ("Chest",        0x0B,  0x08, None,                            'Rupee (Treasure Chest Game)',           ("Gerudo Training Ground MQ", "Master Quest", "Chests",))),
+    ("Gerudo Training Ground MQ Maze Right Side Chest",              ("Chest",        0x0B,  0x08, None,                            'Green Rupee (Treasure Chest Game)',     ("Gerudo Training Ground MQ", "Master Quest", "Chests",))),
```

Comment:
nitpick, but should this be Rupee (1) to match other instances?

Reply:
I think `Rupee (Treasure Chest Game) (1)` would be best as it would work automatically with the starting items code that translates item names in that format into ammo amounts.